build.gradle

```
@@ -113,3 +122,41 @@ subprojects {
         executable "${rootDir}/scripts/docker/lint.sh"
     }
 }
+
+jreleaser {
+    gitRootSearch = true
+
+    signing {
+        active = 'ALWAYS'
+        armored = true
+    }
+
+    deploy {
+        maven {
+            mavenCentral {
+                sonatype {
+                    active = 'ALWAYS'
```

```suggestion
                    active = 'RELEASE'
```

I'm not very sure but the following specifies `RELEASE`:
https://jreleaser.org/guide/latest/examples/maven/maven-central.html#_publishing_snapshots

What do you think?

---

Yeah, I'm also wondering about that point since the two examples (with and without snapshot release settings) are a bit different, e.g., `sonatype` vs. `'release-deploy'`. Anyway, I changed it to the latter example. Will see. Thank you!